src/cockpit/channel.py

```diff
@@ -298,15 +299,14 @@ class ProtocolChannel(Channel, asyncio.Protocol):
     _transport: Optional[asyncio.Transport]
     _loop: Optional[asyncio.AbstractEventLoop]
     _send_pongs: bool = True
-    _last_ping: Optional[Dict[str, object]]
+    _last_ping: Optional[JsonObject]
```

Comment:
.. and here, it does not seem so clear to me when a type should be a JsonDict, and when a JsonObject. This obviously depends on how it gets instantiated, but it feels error-prone, and hard to keep track of which of our objects are JsonObject wrappers and which aren't.